include/tables/hive/TBLSTable.h

```diff
@@ -35,15 +36,33 @@ class TBLSTable : public DBWatchTable<TBLSRow> {
     addColumn("SD_ID");
     addColumn("VIEW_EXPANDED_TEXT");
     addColumn("VIEW_ORIGINAL_TEXT");
+    addColumn("DB_ID");
     addWatchEvent(NdbDictionary::Event::TE_DELETE);
   }
 
   TBLSRow getRow(NdbRecAttr *value[]) {
     TBLSRow row;
     row.mTBLID = value[0]->int64_value();
     row.mSDID = value[1]->int64_value();
+    row.mDBID = value[4]->int64_value();
     return row;
   }
+
+  void removeByDBID(Ndb *conn, Int64 DBID)
+  {
+    AnyMap key;
+    key[4] = DBID;
+    int count = deletesByIndex(conn, "TBLS_N49", key, boost::none);
```

Comment:
Can we add a partition key on these tables

Reply:
same as above